packages/qwik-city/package.json

```diff
@@ -97,9 +97,19 @@
       "require": "./lib/adapters/static/vite/index.cjs"
     },
     "./adapters/vercel-edge/vite": {
-      "types": "./lib/adapters/vercel-edge/vite/index.d.ts",
-      "import": "./lib/adapters/vercel-edge/vite/index.mjs",
-      "require": "./lib/adapters/vercel-edge/vite/index.cjs"
+      "types": "./lib/adapters/vercel/edge/vite/index.d.ts",
```

Comment:
actually, let's give this different types that re-export /edge/vite/index but add `/** @deprecated Please import from adapters/vercel/edge instead */` to all of them